core/src/Streamly/Internal/Data/Fold.hs

```diff
@@ -155,7 +155,7 @@ module Streamly.Internal.Data.Fold
 
     -- * Running A Fold
     , drive
-    , driveBreak
+    , breakStream
```

Comment:
fold & foldBreak
vs
drive & breakStream

foldBreak vs breakStream

Reply:
This seems inconsistent. Does `driveContinue` make sense?

Reply:
We want to specify what is being used to drive the fold e.g. stream or unfold. Depending on that the operation would be breakStream or breakUnfold, just like in stream module we have foldBreak or parseBreak. Also, driveBreak sounded a bit odd. Also, this is internal as of now so we need not worry too much about it.